src/sentry/snuba/metrics/query_builder.py

```diff
@@ -400,6 +401,52 @@ def get_metric_object_from_metric_field(
     return metric_object_factory(op=metric_field.op, metric_mri=metric_field.metric_mri)
 
 
+class AliasMetaType(Enum):
+    TAG = 0
+    DATASET_COLUMN = 1
+    TIME_COLUMN = 2
+    SELECT_METRIC_FIELD = 3
+    GROUP_BY_METRIC_FIELD = 4
+
+
+def get_alias_meta_type(
```

Comment:
could you please check that this function has proper coverage?

Reply:
I will add more tests

Reply:
🙏